ads_plugin.py

```diff
@@ -35,7 +35,7 @@
 
 from AxonDeepSeg.morphometrics.compute_morphometrics import *
```

Comment:
This is not specifically related to this PR, but recently I merged https://github.com/neuropoly/axondeepseg/pull/487 to avoid using wild card imports in the scripts. I guess I might have skipped this script and the specific import (
`from AxonDeepSeg.morphometrics.compute_morphometrics import *`) while we merged my PR. 

Would it be possible to remove the wild card import in this script so that this bad practise is not followed and the codebase is consistent?

Thanks!

Reply:
Yeah I agree we should remove it.